File: lib/tmdb_api/model/production_country.dart
class ProductionCountry {
  String id;
  String name;

  ProductionCountry({this.id, this.name});

  @override
  String toString() {
    return name;
  }

  factory ProductionCountry.fromJson(Map<String, dynamic> json) {
    return ProductionCountry(id: json['iso_3166_1'], name: json['name']);
  }
}

File: android/model/src/main/java/com/tikalk/tmdb/json/model/ProductionCountry.kt
package com.tikalk.tmdb.json.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ProductionCountry(
    @SerialName("iso_3166_1")
    val id: String,

    @SerialName("name")
    val name: String
)